Dockerfile

```diff
@@ -43,6 +45,8 @@ FROM build AS release
 COPY --from=dependencies /telescope/node_modules /telescope/node_modules
 COPY --from=dependencies /telescope/src/frontend/public /telescope/src/frontend/public
 COPY ./src/backend ./src/backend
+# Copy certs into container
+COPY ./certs ./certs
```

Comment:
@raygervais what is the right way for me to get the contents of `./certs` into the container?

We're going to need to do one thing in development, and another in staging, where these files will need to come from another location (i.e., Let's Encrypt).

I did this for now just to test.